packages/@react-aria/tag/src/useTagGroup.ts

```diff
@@ -44,9 +51,9 @@ export function useTagGroup<T>(props: AriaTagGroupProps<T>, state: TagGroupState
   let {direction} = useLocale();
   let keyboardDelegate = props.keyboardDelegate || new TagKeyboardDelegate(state.collection, direction);
   let {gridProps} = useGridList({...props, keyboardDelegate}, state, ref);
+  let {labelProps, fieldProps, descriptionProps, errorMessageProps} = useField(props);
```

Comment:
Noticed that `useGridList` is warning about missing a `aria-label`/`aria-labelledby` in the form stories. You'll want to call this `useField` hook before `useGridList` and supply the `fieldProps` (I think?) to `useGridList`. I wonder if there might be an issue with how `useGridList` uses its own generated `id` and not the user provided one though, don't quite remember why that is if there is even a reason for that at all...

Reply:
Did some more digging, don't think there is a good reason for `useGridList` to not accept a user provided id so you can probably go ahead and make https://github.com/adobe/react-spectrum/blob/c183944ce6a8ca1cf280a1c7b88d2ba393dd0252/packages/%40react-aria/gridlist/src/useGridList.ts#L84 be `let id = useId(props.id);` and pass `fieldProps` to `useGridList`